pyproject.toml

```diff
@@ -238,3 +238,9 @@ enable_error_code = ['explicit-override']
 ignore_missing_imports = true
 warn_redundant_casts = true
 warn_unused_ignores = true
+
+[tool.codespell]
+ignore-words-list = 'nd,dout,nempty'
```

Comment:
There are 3 `dout` and 4 `nempty` in this repository, I think it's better to rename the variables to `d_out` and `n_empty` instead of adding these words to the ignore list.

The help states this list is case sensitive, so I think it's better to add `Nd` and `nD` so we still catch missing letters in "and".